docs/cheatsheet/cheatsheet.html

```diff
@@ -31,13 +31,14 @@
 <div class="header">
 <div class="usage">
 <span class="one">EarlGrey()</span>.selectElementWithMatcher(<span class="two">ViewMatcher</span>)
+                     .indexOf(<span class="three">index</span>)
```

Comment:
Since cheatsheet is for quick reference I feel like we must add usingSearchAction too, what do you think?